react/components/UI/BreadcrumbPath/index.js

```diff
@@ -11,6 +11,7 @@ const Container = styled.div`
   margin: 0 0 2em 0;
   font-size: ${styles.Type.size.h3};
   line-height: ${styles.Type.lineHeight.compact};
+  font-family: ${styles.Type.font.sans};
```

Comment:
Interestingly, most of these React components rely on inherited styles from Ervell. This makes me wonder if we should try and detach that dependency and duplicate styles on the actual components.

Reply:
On base components like buttons/forms/etc I've been just using all: initial. https://github.com/aredotna/ervell/blob/f5dcd4811e8b05542e303042b6be8c74299e6b5e/react/components/UI/GenericButton/index.js#L12 to reset. I'm not terribly concerned about most other things as we'll just fix them as they crop up. But if you want isolation; that's one way to go.

Reply:
If we want to use storybooks, that will probably be how things naturally evolve. Not sure if that's good or bad.

Reply:
Well it's good; we have to kill the old Stylus code. I just mean it'll happen eventually storybooks or not, and if you're building a UI primitive then you'll need to reset using `all: initial` anyway to prevent insane specificity wars from the competing Stylus.